src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.cs

```diff
@@ -61,12 +61,19 @@ public object ScrollOwner
 
 #if __WASM__
 		bool _forceChangeToCurrentView;
-
 		bool IScrollContentPresenter.ForceChangeToCurrentView
 		{
 			get => _forceChangeToCurrentView;
 			set => _forceChangeToCurrentView = value;
 		}
+
+#elif __SKIA__
```

Comment:
Why can't we use `#if __WASM__ || __SKIA__` above?

Reply:
Only to avoid making this property public. 

For WASM, there's an interface `IScrollContentPresenter.cs`, and we do an Explicit implementation to conform to the interface signature.